mesheryctl/internal/cli/root/filter/list.go

```diff
@@ -40,41 +42,36 @@ var listCmd = &cobra.Command{
 	Long:  `Display list of all available filter files.`,
 	Example: `
 // List all WASM filter files present
-mesheryctl filter list	
+mesheryctl filter list	(maximum 25 filters)
+
```

Comment:
Add a comment to suggest that one arg can be passed to search for a filter with specific name. Also mention the need for quotes when specifying an arg with spaces.

Reply:
Page number can be a flag but pagesize can be static like 25 I thought so adding it in the url directly would work

Reply:
@suhail34 Doing so works, but is a lesser practice compared to pulling out configuration items like this separately.

@narharim will you incorporate the comment that @abdullah1308 mentioned?